frontend/console/src/features/modules/ModulesTree.tsx

```diff
@@ -23,7 +23,11 @@ import { classNames } from '../../utils'
 import type { ModuleTreeItem } from './module.utils'
 import { addModuleToLocalStorageIfMissing, listExpandedModulesFromLocalStorage, toggleModuleExpansionInLocalStorage } from './module.utils'
 
-const ExportedIcon = () => <FileExportIcon className='size-4 text-indigo-500 -ml-1' />
+const ExportedIcon = () => (
+  <span className='w-4' title='Exported'>
+    <FileExportIcon className='size-4 text-indigo-500 -ml-1' />
+  </span>
```

Comment:
nit: and mostly curious, if we have a `span` with `w-4` and the icon in it is `size-4`, do we need both? Like can we just put the `size` on the wrapper span?

Also, thanks for adding `title` here! I was thinking the same thing, but didn't get to it yet :)

Reply:
For some reason, the `svg` treats the size as a max size instead of an exact size, so I basically added the span just to force it into a fixed-size container where it can't complain about losing space. My guess is there's something deep inside the implementation of the icon that tries to be "smart" and adjust size automatically, but that's just totally incompatible with complex containers like this. :/